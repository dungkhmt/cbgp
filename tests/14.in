50 50
50 71
22 39
39 33
33 10
10 13
13 24
24 8
8 20
20 35
35 22
42 9
9 40
40 50
50 38
38 19
19 2
2 42
4 37
37 32
32 25
25 16
16 36
36 3
3 11
11 48
48 4
44 41
41 31
31 21
21 29
29 47
47 34
34 18
18 26
26 30
30 43
43 44
14 27
27 46
46 6
6 17
17 1
1 28
28 14
23 45
45 7
7 15
15 12
12 5
5 23
46 21
17 12
23 1
49 5
29 25
40 8
27 7
36 18
37 9
19 3
45 49
22 2
38 20
32 31
30 14
34 6
41 4
47 16
26 11
49 15
42 10
43 48
1
31 7 20 18 0 0 33 15 31 5 26 2 36 3 26 20 34 18 31 19 29 11 29 3 30 20 34 8 31 3 22 0 27 3 23 3 18 18 24 20 48 2 21 19 32 6 28 20 52 0 30 10 39 3 32 8 49 1 32 10 39 11 40 12 27 19 24 2 22 20 20 0 35 17 22 22 25 19 29 23 37 11 32 18 33 11 35 11 34 4 43 2 23 1 31 13 32 4 26 26 
