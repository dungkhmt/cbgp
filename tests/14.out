(21, 38), (38, 32), (32, 9), (9, 12), (12, 23), (23, 7), (7, 19), (19, 34), (34, 21), (41, 8), (8, 39), (39, 49), (49, 37), (37, 18), (18, 1), (1, 41), (3, 36), (36, 31), (31, 24), (24, 15), (15, 35), (35, 2), (2, 10), (10, 47), (47, 3), (43, 40), (40, 30), (30, 20), (20, 28), (28, 46), (46, 33), (33, 17), (17, 25), (25, 29), (29, 42), (42, 43), (13, 26), (26, 45), (45, 5), (5, 16), (16, 0), (0, 27), (27, 13), (22, 44), (44, 6), (6, 14), (14, 11), (11, 4), (4, 22), (45, 20), (16, 11), (22, 0), (48, 4), (28, 24), (39, 7), (26, 6), (35, 17), (36, 8), (18, 2), (44, 48), (21, 1), (37, 19), (31, 30), (29, 13), (33, 5), (40, 3), (46, 15), (25, 10), (48, 14), (41, 9), (42, 47), 
0: (31, 7), 1: (20, 18), 2: (0, 0), 3: (33, 15), 4: (31, 5), 5: (26, 2), 6: (36, 3), 7: (26, 20), 8: (34, 18), 9: (31, 19), 10: (29, 11), 11: (29, 3), 12: (30, 20), 13: (34, 8), 14: (31, 3), 15: (22, 0), 16: (27, 3), 17: (23, 3), 18: (18, 18), 19: (24, 20), 20: (48, 2), 21: (21, 19), 22: (32, 6), 23: (28, 20), 24: (52, 0), 25: (30, 10), 26: (39, 3), 27: (32, 8), 28: (49, 1), 29: (32, 10), 30: (39, 11), 31: (40, 12), 32: (27, 19), 33: (24, 2), 34: (22, 20), 35: (20, 0), 36: (35, 17), 37: (22, 22), 38: (25, 19), 39: (29, 23), 40: (37, 11), 41: (32, 18), 42: (33, 11), 43: (35, 11), 44: (34, 4), 45: (43, 2), 46: (23, 1), 47: (31, 13), 48: (32, 4), 49: (26, 26), 
31 7 20 18 0 0 33 15 31 5 26 2 36 3 26 20 34 18 31 19 29 11 29 3 30 20 34 8 31 3 22 0 27 3 23 3 18 18 24 20 48 2 21 19 32 6 28 20 52 0 30 10 39 3 32 8 49 1 32 10 39 11 40 12 27 19 24 2 22 20 20 0 35 17 22 22 25 19 29 23 37 11 32 18 33 11 35 11 34 4 43 2 23 1 31 13 32 4 26 26 
intersectionCount=0
minAngle=0.244979
sumAngle=65.7956
minEdgeLength=1.41421
sumEdgeLength=370.243
minDistance=1
sumDistance=43048.8

0: (31, 7)
1: (20, 18)
2: (0, 0)
3: (33, 15)
4: (31, 5)
5: (26, 2)
6: (36, 3)
7: (26, 20)
8: (34, 18)
9: (31, 19)
10: (29, 11)
11: (29, 3)
12: (30, 20)
13: (34, 8)
14: (31, 3)
15: (22, 0)
16: (27, 3)
17: (23, 3)
18: (18, 18)
19: (24, 20)
20: (48, 2)
21: (21, 19)
22: (32, 6)
23: (28, 20)
24: (52, 0)
25: (30, 10)
26: (39, 3)
27: (32, 8)
28: (49, 1)
29: (32, 10)
30: (39, 11)
31: (40, 12)
32: (27, 19)
33: (24, 2)
34: (22, 20)
35: (20, 0)
36: (35, 17)
37: (22, 22)
38: (25, 19)
39: (29, 23)
40: (37, 11)
41: (32, 18)
42: (33, 11)
43: (35, 11)
44: (34, 4)
45: (43, 2)
46: (23, 1)
47: (31, 13)
48: (32, 4)
49: (26, 26)

Time taken: 0.0402925 s
